core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -113,7 +114,8 @@ func FilterName(addr common.Address) string {
 
 func (f *FwdMgr) ForwarderFor(addr common.Address) (forwarder common.Address, err error) {
 	// Gets forwarders for current chain.
-	fwdrs, err := f.ORM.FindForwardersByChain(utils.Big(*f.evmClient.ChainID()))
+	chainId := f.evmClient.ConfiguredChainID()
+	fwdrs, err := f.ORM.FindForwardersByChain(utils.Big(*chainId))
```

Comment:
nit: chainId value is never used elsewhere so we can probably merge this:

```suggestion
	fwdrs, err := f.ORM.FindForwardersByChain(utils.Big(*f.evmClient.ConfiguredChainID()))
```